src/Generator.cc

```diff
@@ -149,7 +149,7 @@ bool Generator::Generate(const FileDescriptor *_file,
 
     auto ns = getNamespaces(_file->package());
 
-    for (auto name : ns)
```

Comment:
loop variable is copied but only used as const reference; consider making it a const reference